snaps/features/custom-ui/dialogs.md

```diff
@@ -122,6 +123,104 @@ const walletAddress = await snap.request({
 <img src={require("../../assets/prompt-dialog.png").default} alt="Prompt dialog example" width="360px" style={{border: "1px solid #DCDCDC"}} />
 </p>
 
+## Display a custom dialog
```

Comment:
Can you update "What's new?" linking to the new content?

Reply:
Done!